docs/en/deploy/Running-Alluxio-On-Docker.md

```diff
@@ -396,6 +396,62 @@ You can find more details about the worker storage [here]({{ '/en/core-services/
 
 ### Enable POSIX API access
 
+Alluxio POSIX access is implemented via FUSE.
+[POSIX API docs]({{ '/en/api/POSIX-API.html' | relative_url }}) provides more details about how to configure Alluxio POSIX API.
+There are two options to enable POSIX access to Alluxio in a docker environment.
+
++ Option1: Enable FUSE support when running a worker container, or
++ Option2: Run a standalone Alluxio FUSE container.
+
+Enabling FUSE on a worker container may yield easier deployment and higher performance
+as it only requires setting a property `alluxio.worker.fuse.enabled=true`,
```

Comment:
alluxio.worker.fuse.mount.point is also required which doesn't have a default value.
By the way, do we wat a default fuse mount point?

Reply:
yeah, I think we can have `/mnt/alluxio-fuse` as the default mount point,
in this way the doc can be a bit simpler